lib/Packager/module-resolver.es6

```diff
@@ -375,8 +375,8 @@ function EXPAND (target, log, cache, opts, cwd) {
 		if(result.json) {
 			let moduleDir = result.json.moduleDir;
 			if (!moduleDir) {
-				let libDir = path.join(opts.cwd, opts.libDir, dir);
-				if (libDir === step) {
+				let libDir = path.resolve(step, '..')
```

Comment:
This may work for the case you were facing but I fear about it's overall applicability. I haven't been able to test it but at first glance this appears incorrect in _all_ cases.